frontends/concrete-python/concrete/fhe/compilation/configuration.py

```diff
@@ -616,6 +616,115 @@ def _promotions(
         return required_x_bit_width, required_y_bit_width
 
 
+class MultivariateStrategy(str, Enum):
+    """
+    MultivariateStrategy, to specify implementation preference for multivariate operations.
+    """
+
+    PROMOTED = "promoted"
```

Comment:
Happy to see the work for comparison generalize well other operation (casted/promoted)

Reply:
Agreed.